src/components/rules-creation-page/SyncRulesWizard.jsx

```diff
@@ -43,6 +44,14 @@ class SyncRulesWizard extends React.Component {
             description: undefined,
             help: undefined,
         },
+        {
+            key: "scheduler",
+            label: i18n.t("Scheduling"),
+            component: SchedulerStep,
+            validationKeys: ["frequency", "enabled"],
+            description: i18n.t("Configure the scheduling frequency for the synchronization rule"),
```

Comment:
Maybe add a link to documentation about the format of the cron expression? In vaccination-app we use react-linkify to render links easily, but feel free to do it how you see fit (if we really need a link here!)

Reply:
I think it's a good idea to add a link, but not sure to what page we should reference about cron expressions. We only support a subset of them (6 values including seconds) to make it more simple.